src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -6,6 +6,13 @@
     <BuildInfoCacheDir>$(ToolsDir)BuildInfoCache/</BuildInfoCacheDir>
   </PropertyGroup>
 
+  <!-- For backward compatibility, Include XmlUpdateSteps as Xml-type updaters. -->
```

Comment:
Sure. Filed https://github.com/dotnet/core-eng/issues/428 for now.

@dleeapho How should we file BuildTools breaking changes? I suspect this will be more manageable when we get a release model.